Test/UnitTests/UnitTests.csproj

```diff
@@ -26,13 +26,8 @@
     <Reference Include="System" />
     <Reference Include="System.Xml" />
     <Reference Include="System.Core" />
-    <PackageReference Include="NUnit">
-      <Version>3.9.0</Version>
-    </PackageReference>
-    <PackageReference Include="NUnit3TestAdapter">
-      <Version>3.10.0</Version>
-    </PackageReference>
-    <PackageReference Include="Mono.Cecil" Version="0.10.0" />
```

Comment:
Cecil not required?

Reply:
Yeah doesn't seem to be needed:
```
C:\xmd\main\external\mono-addins\Test\UnitTests>refdump C:\xmd\main\external\mono-addins\Test\UnitTests\bin\Debug\UnitTests.dll
UnitTests, Version=0.0.0.0, Culture=neutral, PublicKeyToken=0738eb9f132ed756
References:
Mono.Addins, Version=1.0.0.0, Culture=neutral, PublicKeyToken=0738eb9f132ed756
Mono.Addins.Setup, Version=1.0.0.0, Culture=neutral, PublicKeyToken=0738eb9f132ed756
mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089
nunit.framework, Version=3.9.0.0, Culture=neutral, PublicKeyToken=2638cd05610744eb
System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089
System.Core, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089
System.Xml, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089```

Reply:
It should be there, since we depend on CecilReflector.

Reply:
Maybe CecilReflector imports it transitively.